pkg/scaffold/role.go

```diff
@@ -58,4 +58,10 @@ rules:
   - statefulsets
   verbs:
   - "*"
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+	verbs:
+  - "*"
```

Comment:
we might to only want to grant access to create and maybe get? I don't think that we should be able to update all service monitors. Thoughts?

Reply:
Yes that sgtm. 👍